.evergreen/run-atlas-tests.sh

```diff
@@ -2,9 +2,12 @@
 
 set -o errexit  # Exit the script with error if any of the commands fail
 
-source "${PROJECT_DIRECTORY}/.evergreen/init-node-and-npm-env.sh"
+if test -f secrets-export.sh; then
+	source secrets-export.sh
```

Comment:
```suggestion
  source secrets-export.sh
```
Just a nit on formatting.